code/game/machinery/bots/ed209bot.dm

```diff
@@ -753,14 +753,14 @@ Auto Patrol: []"},
 		if(perp.dna && perp.dna.mutantrace && perp.dna.mutantrace != "none")
 			threatcount += 2
 
-		if(!perp.wear_id)
+		if(!perp.get_visible_id())
 			if(idcheck)
 				threatcount += 4
 			else
 				threatcount += 2
 
 		//Agent cards lower threatlevel.
-		if(perp.wear_id && istype(perp.wear_id.GetID(), /obj/item/weapon/card/id/syndicate))
+		if(perp.get_visible_id() && istype(perp.get_visible_id(), /obj/item/weapon/card/id/syndicate))
```

Comment:
It isn't critical (or really even particularly important) in this case, but as a general rule it's a better idea to call a proc once and assign the return value to a variable if you want to check the return value multiple times.